core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -44,6 +59,18 @@
     @Override
     public void create() {
         pixmap = new Pixmap(WIDTH, HEIGHT, Pixmap.Format.RGBA8888);
+        stage = new Stage();
+        createBasicSkin();
+        
+        try {
+            logoTexture = new Texture(Gdx.files.internal("./core/src/de/adesso/brainysnake/img/BrainySnake_Headline.png"));
```

Comment:
The header Image png should be moved to core assets folder (it´s a common asset). Then you can load it with _new Texture("[path within assets folder]BrainySnake_Headline.png");_

Reply:
Moved the image to the the assets folder and changed the loading.